src/app.js

```diff
@@ -0,0 +1,29 @@
+import './assets/scss/app.scss';
+import $ from 'cash-dom';
+
+
+export class App {
+  initializeApp() {
+    let self = this;
+
+    $('.load-username').on('click', function (e) {
+      let userName = $('.username.input').val();
+
+      fetch('https://api.github.com/users/' + userName)
+        .then((response)=> {response.json})
+        .then(function (body) {
+          self.profile = body;
+          self.update_profile();
+        })
+
+    })
+
+  }
+
+  update_profile() {
+    $('#profile-name').text($('.username.input').val())
```

Comment:
```suggestion
    $('#profile-name').text(this.profile.login);
```

instead of getting value from input better way from data, there may be a situation when profile can be not found